src/main/java/mate/academy/hibernate/relations/dao/impl/MovieDaoImpl.java

```diff
@@ -12,11 +15,33 @@
 
     @Override
     public Movie add(Movie movie) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
```

Comment:
Consider using try-with-resources for managing the `Transaction` to ensure it is closed properly, even if exceptions occur. This will help maintain resource integrity and reduce boilerplate code.